sprocs/group_info.sql

```diff
@@ -1,11 +1,14 @@
 CREATE FUNCTION
     group_info (
-        IN assessment_id bigint
+        IN assessment_id bigint,
+        IN course_instance_id bigint DEFAULT null
     ) RETURNS TABLE (
         id bigint,
         name text,
         uid_list text[],
-        user_name_list text[]
+        user_name_list text[],
+        user_id_list bigint[],
+        roles text[]
```

Comment:
Nit: consider calling this `user_roles_list` instead for consistency with the other output values? Tbh, I never liked the `_list` suffix, so I'd also be happy if you refactored to remove that 🙂

Reply:
I think I'll refactor it as `user_roles_list` for now. Perhaps we can remove the `_list` suffixes in another PR.